test/fixtures/incorrect.js

```diff
@@ -193,10 +196,10 @@ noop(oneVar1);
 noop(oneVar2);
 
 // `one-var-declaration-per-line`.
-const oneVar1 = 'foo'; const oneVar2 = 'bar';
+const oneVar3 = 'foo'; const oneVar4 = 'bar';
```

Comment:
Having these variables start counting at 3 is weird. Follow another naming pattern maybe?